### PR TITLE
Get user session guide

### PR DESCRIPTION
**Pull request by abdulajet**
Added a user session guide for JS and android

Page to review: https://nexmo-developer-pr-3886.herokuapp.com/client-sdk/in-app-messaging/guides/get-user-sessions/javascript

### CODE REVIEW

**Comment by pardel** [comment]
Are the session calls based on user ID rather than username?! This is probably the first time it happens...

**Comment by conshus** [comment]
> Are the session calls based on user ID rather than username?! This is probably the first time it happens...

Do you mean like the actual key name `user_id`? Yeah, I think this is the first time. Previously, for like getUser, you just pass in an `id` like getUser(id), no object or anything. 

I don't think I've seen any `username` keys. There's `name` and `display_name` when creating a user on the server.

File: _documentation/en/client-sdk/in-app-messaging/guides/get-user-sessions.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Get User Sessions
+navigation_weight: 10
+---
+
+# Get User Sessions
+
+## Overview
+
+This guide covers how to get the a user's active [sessions](/conversation/concepts/session), which can be used as a way to show an online status.
```
Comment (conshus) [inline _documentation/en/client-sdk/in-app-messaging/guides/get-user-sessions.md:10]:
```suggestion
This guide covers how to get a user's active [sessions](/conversation/concepts/session), which can be used as a way to show their online status. 
```

File: _documentation/en/client-sdk/in-app-messaging/guides/get-user-sessions.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+navigation_weight: 10
+---
+
+# Get User Sessions
+
+## Overview
+
+This guide covers how to get the a user's active [sessions](/conversation/concepts/session), which can be used as a way to show an online status. 
+
+Before you begin, make sure you [added the SDK to your app](/client-sdk/setup/add-sdk-to-your-app) and you are able to [create a conversation](/client-sdk/in-app-messaging/guides/simple-conversation).
```
Comment (conshus) [inline _documentation/en/client-sdk/in-app-messaging/guides/get-user-sessions.md:12]:
```suggestion
Before you begin, make sure you [added the SDK to your app](/client-sdk/setup/add-sdk-to-your-app) and are able to [create a conversation](/client-sdk/in-app-messaging/guides/simple-conversation).
```

**Review by conshus — CHANGES_REQUESTED** [review]
Looks really good. Just a couple of things I noted.
Also, I see that there are `Note` in some of the guides about the tutorial for the chat application that the chat app doesn't cover. Do we want to add them to the chat app or remove the note?

**Comment by abdulajet** [comment]
> Also, I see that there are `Note` in some of the guides about the tutorial for the chat application that the chat app doesn't cover. Do we want to add them to the chat app or remove the note?

I am not sure what you mean by this, like the chat app doesn't have the feature in the guide?

**Comment by conshus** [comment]
> > Also, I see that there are `Note` in some of the guides about the tutorial for the chat application that the chat app doesn't cover. Do we want to add them to the chat app or remove the note?
> 
> I am not sure what you mean by this, like the chat app doesn't have the feature in the guide?

Yeah, exactly. We are linking them to a tutorial that doesn't showcase the feature we are talking about on the page.

**Review by conshus — APPROVED** [review]
Everything looks good to me.

**Pull request merged**